src/kemal/config.cr

```diff
@@ -24,6 +24,8 @@ module Kemal
       @error_handler = nil
       @always_rescue = true
       @server = uninitialized HTTP::Server
+      @dir_listing = false
+      @enable_gzip = false
```

Comment:
Perhaps we could change 'serve_static' from a bool to a config hash? This could include options for gzip, dir_listing etc..?
```crystal
serve_static = {"gzip" => true, "dir_listing" => true }
```